letsdrill.rb

```diff
@@ -1,24 +1,31 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  if integer >= 90
+    return "A"
+  elsif integer >= 80
+    return "B"
+  elsif integer >= 70
+    return "C"
+  elsif intger >= 60
+    return "D"
+  else
+    return "F"
```

Comment:
These explicit `returns` aren't required. Rubyists tend to avoid explicit calls to `return` unless required (by a guard clause, for instance).

```ruby
...
    "D"
  else
    "F"
  end
end
```